src/utils.rs

```diff
@@ -15,11 +15,7 @@
 
 */
 
-#[cfg(not(feature = "std"))]
```

Comment:
No need to use alloc in `no_std` only. `std::string::String` points to `alloc::string::String` as well. So I removed this completely.